### DISCUSSION TITLE
UI Problem with the annotation tool

### DISCUSSION THREAD

**Discussion by tkunanuntakij** [sebastian-hofstaetter/teaching#53] [Advanced IR: Exercise Q&A] [answered]
As in the picture, the UI is a bit broken, it trims a bit on the right most text and the menu went out of the screen.
I access the site, full screen, with `Microsoft Edge Version 99.0.1150.55` on Window.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/1088880/160464872-18f6deea-c6ea-403f-978a-701b69ead4ec.png">

**Comment by sophiaalthammer** [comment, marked as answer]
Hi,
we are sorry about the UI interface, this problem could appear because of the long text, which we didn't have in the previous annotation exercises. we would suggest using another browser or trying to annotate on your phone, maybe you dont get those problems there? I am annotating in Firefox and there I don't get those problems so far.

**Reply by ConBoe** [reply]
Hi, 
I have had the same problem with the UI in Chrom. I then installed and used Firefox but the issue is still there for me
Just wanted to let you know

Edit: I also am a Windows 10 user
